plugins/scaffolder/src/components/fields/OwnedGroupsPicker/OwnedGroupsPicker.tsx

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { identityApiRef, useApi } from '@backstage/core-plugin-api';
+import useAsync from 'react-use/lib/useAsync';
+import { EntityPicker } from '../EntityPicker/EntityPicker';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import { OwnedGroupsPickerProps } from './schema';
+import { TextField } from '@material-ui/core';
+
+export { OwnedGroupsPickerSchema } from './schema';
+
+export const OwnedGroupsPicker = (props: OwnedGroupsPickerProps) => {
+  const {
+    schema: { title = 'Group', description = 'A group from the catalog' },
+    uiSchema,
+    required,
+  } = props;
+
+  const identityApi = useApi(identityApiRef);
+  const { loading, value: identityRefs } = useAsync(async () => {
+    const identity = await identityApi.getBackstageIdentity();
+    return identity.userEntityRef;
```

Comment:
So I think this would be more of a `GroupsImPartOfPicker`, not ownership as such. It could be implemented that way in the auth-backend, but I would recommend using the `ownershipEntityRefs` from `identity` as the groups, and then you don't need the entity picker at all, you can just render a dropdown select with these options.

You might want to go and make a request to the catalog for each of the refs in the `ownershipEntityRefs` however to go get their `metadata.title` etc to display in a nice way.

Reply:
Also to be clear, maybe there is a use case for this picker, so maybe we could also have the `OwnedGroupsPicker` one and also the `UsersMemberOfGroup` picker (names need work) :joy:

Reply:
Did a ..small.. refactor on this but still working on it 😅

Reply:
Hey @benjdlambert,

Could I get another sanity check on this to just make sure I am moving in the right direction?

Thanks!